src/components/layout/Header.js

```diff
@@ -0,0 +1,66 @@
+import { openCartModal } from "../cart/CartModal.js";
+import cartStore from "../../store/cartStore.js";
+
+function Header(title = "쇼핑몰") {
```

Comment:
이런 경우 props 처럼 객체 형태로 넘겨보면 어떨까요?

```suggestion
function Header({ title = "쇼핑몰" }) {
```